ui/raidboss/emulator/ui/EmulatedPartyInfo.js

```diff
@@ -286,10 +344,37 @@ export default class EmulatedPartyInfo extends EventBus {
         false);
   }
 
-  _wrapCollapse(label, $obj, onclick) {
+  _wrapCollapse(time, name, icon, text, classes, $obj, onclick) {
```

Comment:
I'm not sure how that would improve readability where the function is called, since the object would have to be constructed there anyways? I guess I'm just not seeing the benefit here since it's only called from three locations.

Maybe it would make more sense to make this a helper class that could chain function calls instead, such that calls became:
```js
let wrapped = 
    new WrapCollapse($initDataViewer)
    .setTime('00:00')
    .setName('Initial Data')
    .setClasses('data')
    .setCallback(() => {
      $initDataViewer.textContent = JSON.stringify(per.initialData, null, 2);
    }))
    .get();
```

Reply:
Oh, the readability is just from sheer number of parameters, especially with multiple duplicate nulls.  I meant more:

```javascript
    $container.append(this._wrapCollapse({
        time: '00:00',
        name: 'Initial Data',
        icon: null, // optional??
        text: null,
        classes: ['data'],
        callback: $initDataViewer, () => {
          $initDataViewer.textContent = JSON.stringify(per.initialData, null, 2);
        },
    }));
```

It sounds like you disagree about readability though?

Reply:
Switched to object for latest commit.